src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -27,7 +28,9 @@ export class HuiGraphHeaderFooter extends LitElement
 
   @property() public hass?: HomeAssistant;
   @property() protected _config?: GraphHeaderFooterConfig;
-  @property() private _coordinates?: any;
+  @property() private _coordinates?: number[][];
+
+  private _loadingInitialData: boolean = true;
```

Comment:
Instead of doing this we should update the `coordinates` function. Instead of returning undefined when there is no state history we should return an empty array. That way we don't need an extra variable. 

If `_coordinates` is undefined we know we haven't updated from the state history yet. But if the Length is of coordinates is 0 then no state history found

Reply:
@bramkragten What do you think of this?

Reply:
Or use undefined and null